### PR TITLE
removed isCilium flag

### PR DESCRIPTION
**Pull request by sdsen**
Addresses #45

### CODE REVIEW

File: run1.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 python3 src/annotator.py -o op/katran/txl_katran -s examples/katran -c op/katran/commented_katran -t op/katran/katran.function_file_list.json -u op/katran/katran.struct_file_list.json -p katran -d projects/katran/boston_katran_comments.json
-python3 src/annotator.py -o op/cilium/txl_cilium -s examples/cilium -c op/cilium/commented_cilium -t op/cilium/cilium.function_file_list.json -u op/cilium/cilium.struct_file_list.json --isCilium -p cilium -d projects/cilium/boston_cilium_comments.json
+python3 src/annotator.py -o op/cilium/txl_cilium -s examples/cilium -c op/cilium/commented_cilium -t op/cilium/cilium.function_file_list.json -u op/cilium/cilium.struct_file_list.json -p cilium -d projects/cilium/boston_cilium_comments.json
```
Comment (palanik1) [inline run1.sh:2]: @sdsen can you move this change to run_annotator.sh

**Review by palanik1 — CHANGES_REQUESTED** [review]

File: run1.sh (hunk 1 above)
Comment (sdsen) [reply run1.sh:2]: Fixed.

**Review by palanik1 — APPROVED** [review]
lgtm

**Comment by palanik1** [comment]
@sdsen can you squash and push..Right now is merging is blocked on needing all commits to be signed.

**Pull request merged**